### PR TITLE
Migrated Batch 5 of Thermal Conductivity

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

File: src/data/thermal-conductivity-constants.ts [hunk 1]
```diff
@@ -2050,6 +2050,730 @@ export const DIPPRThermalConductivityDictionary: Record<
...
+    minimumTemperature: 189.79,
+    maximumTemperature: 394.45,
+    thermalConductivityAtMinimumTemperature: 0.5452,
+    thermalConductivityAtMaximumTemperature: 0.3227,
+  },
+
+  HydrogenSulfide: {
+    name: 'Hydrogen sulfide',
+    formula: 'H2S',
+    CAS: '04-06-7783',
```
Comment (maneetgoyal) [inline src/data/thermal-conductivity-constants.ts:2467]:
```suggestion
    CAS: '7783-06-04',
```

Be more mindful of these mistakes.

File: src/data/thermal-conductivity-constants.ts [hunk 2]
```diff
@@ -2050,6 +2050,730 @@ export const DIPPRThermalConductivityDictionary: Record<
...
+  },
+
+  Methane: {
+    name: 'Methane',
+    formula: 'CH4',
+    CAS: '74-82-8',
+    molecularWeight: 16.0425,
+    C1: 0.41768,
+    C2: -0.0024528,
+    C3: 3.56e-6,
```
Comment (maneetgoyal) [inline src/data/thermal-conductivity-constants.ts:2536]:
```suggestion
    C3: 3.5588e-6,
```

Don't loose precision in data.

**Review by maneetgoyal — APPROVED** [review]
Minor mistakes

File: src/data/thermal-conductivity-constants.ts [hunk 3]
```diff
@@ -2050,6 +2050,730 @@ export const DIPPRThermalConductivityDictionary: Record<
...
+    C1: 0.20348,
+    C2: -0.0003106,
+    minimumTemperature: 183.45,
+    maximumTemperature: 302.15,
+    thermalConductivityAtMinimumTemperature: 0.1465,
+    thermalConductivityAtMaximumTemperature: 0.1096,
+  },
+
+  MethylButyrate: {
+    name: 'MethylButyrate',
```
Comment (sumitnegi7) [inline src/data/thermal-conductivity-constants.ts:2766]:
```suggestion
    name: 'Methyl butyrate',
```

**Review by sumitnegi7 — APPROVED** [review]

**Review by saikat-samanta — APPROVED** [review]

**Pull request merged**